src/router/schema.ts

```diff
@@ -11,7 +11,13 @@ import config from '../config';
 import * as kafka from '../kafka';
 
 export async function composeLatest(req, res) {
-	const schema = await getAndValidateSchema(connection);
+	const { validate } = Joi.attempt(
+		req.query,
+		Joi.object().keys({
+			validate: Joi.boolean().default(true).allow(''),
```

Comment:
I suppose we can have `GET /schema/latest` to have this param, but docs should explicitly tell that this endpoint sacrifices guaranteed validity in favor of simplicity. 

The edge case here is some federated `service A` rollback to older version, which will still serve previous (latest) version, thus `/schema/latest` is not 100% correct

Reply:
Actually, yes, as you said, maybe `/schema/latest` even should not validate schema by default, so that we would drop this param entirely. Then I can release this version as major, since its a breaking change